Python/Product/PythonTools/PythonTools/Project/PythonFileNode.cs

```diff
@@ -190,7 +190,7 @@ internal PythonFileNode(CommonProjectNode root, ProjectElement e)
         }
 
         private VsProjectAnalyzer GetAnalyzer() {
-            return ((PythonProjectNode)ProjectMgr).GetAnalyzer();
+            return ((PythonProjectNode)ProjectMgr).TryGetAnalyzer();
```

Comment:
Now that `GetAnalyzer()` may return `null`, there's going to be an issue with the implementation of `GetAnalysisEntry` right below. It's not expecting to get a `null` analyzer. Are there any callers left for `GetAnalysisEntry` or did you remove them all?

Reply:
i would rename this function to `TryGetAnalyzer`.